src/components/track/mobile/TrackList.tsx

```diff
@@ -105,19 +109,19 @@ const TrackList = ({
           uid={track.uid}
           isDeleted={track.isDeleted}
           onSave={onSave}
+          onRemove={onRemove}
           togglePlay={togglePlay}
           trackItemAction={trackItemAction}
           isReorderable={isReorderable}
           isDragging={isDragging}
         />
       </div>
     )
+    const key = track?.time
+      ? `${track.trackId}:${track.time}`
+      : track.trackId.toString()
     return isReorderable ? (
-      <Draggable
-        key={track.trackId}
-        draggableId={`${track.trackId}`}
-        index={idx}
-      >
+      <Draggable key={key} draggableId={key} index={idx}>
```

Comment:
@raymondjacobson I was going to put this in a separate PR, but this is a bug where if there are duplicate tracks in a playlist, then dragging one of them hides all of them. 
lmk what you think of this solution. I figure all playlist tracks have a unique time key which makes them unique, so I used that here.